src/styles/styled.ts

```diff
@@ -1,35 +1,38 @@
-import type * as Stitches from 'stitches-native';
 import { StyleSheet } from 'react-native';
+import type * as Stitches from 'stitches-native';
 import { createStitches } from 'stitches-native';
 
 import * as utils from './utils';
-import * as typography from '~design-system/typography';
+
 import * as colors from '~design-system/colors';
-import * as space from '~design-system/spacing';
 import * as radii from '~design-system/radii';
+import space from '~design-system/spacing.json';
+import * as typography from '~design-system/typography';
 import * as designSystemUtils from '~design-system/utils';
 
 const { styled, css, createTheme, config, theme, useTheme, ThemeProvider } =
   createStitches({
     theme: {
-      colors: colors.light,
+      colors: utils.transformColors(colors),
```

Comment:
Move `transformColors` to `designSystemUtils` since now it is accidentally being passed to the Stitches utils as well.